src/script/ismine.h

```diff
@@ -19,7 +19,8 @@ class CScript;
 enum isminetype
 {
     ISMINE_NO = 0,
-    //! Indicates that we don't know how to create a scriptSig that would solve this if we were given the appropriate private keys
```

Comment:
why removing this comment? I would rather keep it.

Reply:
thanks, fixed.